### PR TITLE
Adding dockerfile for running the web UI for Satellite

### PR DESCRIPTION
**Pull request by phutchins**

### CODE REVIEW

File: satellite/peer.go [hunk 1]
```diff
@@ -181,6 +181,7 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config *Config) (*
 	var err error
 
 	{ // setup listener and server
+		fmt.Println("Starting listener and server")
```
Comment (kaloyan-raev) [inline satellite/peer.go]: We have to remove these `fmt.Println`s before merging.

File: cmd/satellite/Dockerfile [hunk 2]
```diff
@@ -2,15 +2,30 @@ ARG CGO_ENABLED=1
...
 EXPOSE 7777
+EXPOSE 10100
 WORKDIR /app
 VOLUME /root/.local/share/storj/satellite
+COPY /web/satellite /app
 COPY --from=build-env /app /app/satellite
-COPY cmd/satellite/entrypoint /entrypoint
+COPY /cmd/satellite/entrypoint /entrypoint
+COPY --from=satellite-ui /app/dist /app/dist
+COPY --from=satellite-ui /app/static /app/static
```
Comment (jtolds) [inline cmd/satellite/Dockerfile]: nice, glad this worked

File: cmd/satellite/Dockerfile [hunk 3]
```diff
@@ -2,15 +2,30 @@ ARG CGO_ENABLED=1
...
     CONF_PATH=/root/.local/share/storj/satellite \
-    BOOTSTRAP_ADDR=bootstrap.storj.io:8080
+    BOOTSTRAP_ADDR=bootstrap.storj.io:8080 \
+    STORJ_CONSOLE_STATIC_DIR=/app \
+    STORJ_CONSOLE_ADDRESS=0.0.0.0:10100
 EXPOSE 7777
+EXPOSE 10100
 WORKDIR /app
 VOLUME /root/.local/share/storj/satellite
+COPY /web/satellite /app
```
Comment (jtolds) [inline cmd/satellite/Dockerfile]: wait, why is this needed?

File: cmd/storagenode/entrypoint [hunk 4]
```diff
@@ -1,6 +1,14 @@
 #!/bin/sh
 set -euo pipefail
 
+if [[ ! -f "${IDENT_PATH}/identity.key" ]]; then
+  cd /tmp
+  wget https://storj-v3-alpha-builds.storage.googleapis.com/e82d336-go1.11/identity_linux_amd64.zip
+  unzip identity_linux_amd64.zip
+  mv identity_linux_amd64 identity
+  /tmp/identity create --difficulty 5 storagenode
```
Comment (jtolds) [inline cmd/storagenode/entrypoint]: this concerns me for three reasons -

first, didn't we just copy the identity binary we compiled in?
secondly, this is way too low of a difficulty and identities shouldn't get generated automatically (ssh keys aren't).
third, if this works, this implies that we're not requiring signed certificates

File: cmd/satellite/Dockerfile (hunk 3 above)
Comment (stefanbenten) [reply cmd/satellite/Dockerfile]: you need the entire folder to have the frontend working
Thats what Jens and me got told from the Kiev Team

File: cmd/storagenode/entrypoint (hunk 4 above)
Comment (stefanbenten) [reply cmd/storagenode/entrypoint]: I think Bender is on it to do exactly that.

File: satellite/console/consoleweb/server.go [hunk 5]
```diff
@@ -62,6 +63,8 @@ func NewServer(logger *zap.Logger, config Config, service *console.Service, list
 		listener: listener,
 	}
 
+	log.Printf("Starting Satellite UI...")
```
Comment (kaloyan-raev) [inline satellite/console/consoleweb/server.go]: You can use the `logger` that was passed to the constructor, i.e. `logger.Debug(...)`.

File: satellite/peer.go [hunk 6]
```diff
@@ -181,6 +181,7 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config *Config) (*
 	var err error
 
 	{ // setup listener and server
+		zap.S().Debug("Starting listener and server")
```
Comment (kaloyan-raev) [inline satellite/peer.go]: Sorry, I've just noticed the there is a logger `log` passed to the constructor. So the corrector logging should be done via `log.Debug(...)`.

File: cmd/storagenode/entrypoint (hunk 4 above)
Comment (phutchins) [reply cmd/storagenode/entrypoint]: Updated default difficulty to 30 with a way to configure it. This enables testing in dev environment until we take the time to add high difficulty certs and enforce signing.

File: satellite/console/consoleweb/server.go (hunk 5 above)
Comment (phutchins) [reply satellite/console/consoleweb/server.go]: Updated.

Comment (kaloyan-raev) [reply satellite/console/consoleweb/server.go]: There is no `s` at this point. It should be `logger.Debug(...)` instead of `s.log.Debug(...)`.

File: satellite/console/consoleweb/server.go [hunk 7]
```diff
@@ -6,6 +6,7 @@ package consoleweb
 import (
 	"context"
 	"encoding/json"
+	"log"
```
Comment (kaloyan-raev) [inline satellite/console/consoleweb/server.go]: This import has to be removed as it is not used.

File: satellite/peer.go [hunk 8]
```diff
@@ -181,6 +181,7 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config *Config) (*
 	var err error
 
 	{ // setup listener and server
+		log.Debug("Starting listener and server")
```
Comment (egonelbre) [inline satellite/peer.go]: Any particular reason these log lines were added? The following lines aren't actually starting them, they are just setting them up. Starting happens in a different place.

Comment (kaloyan-raev) [reply satellite/peer.go]: @phutchins needed them to debug some issue and wanted to keep them for future debugging too.

Comment (stefanbenten) [reply satellite/peer.go]: We encountered issues where the setup/startup hung due to incorrect paths in the config file.
As the satellite didnt print out at which step it was, it was very difficult to troubleshoot
